stripe/src/main/java/com/stripe/android/model/PaymentIntent.kt

```diff
@@ -354,10 +344,16 @@ data class PaymentIntent internal constructor(
     }
 
     companion object {
+        private const val clientSecretRegexPattern = "^pi_[^_]+_secret_[^_]+$"
```

Comment:
I have seen cases in the past where the token regex is not as I would expect.   For instance I think sometimes for merchant ids it can be acct_, but there are other things possible as well.  Is this a rule or is this just what we have observed so far?

Reply:
Removed